app/controllers/computacenter/api/base_controller.rb

```diff
@@ -12,9 +12,17 @@ class Computacenter::API::BaseController < ApplicationController
 
 private
 
+  def identify_user!
+    @user = APIAuthenticationService.identify_user(bearer_token)
+  end
+
+  def bearer_token
+    request.headers['Authorization'].to_s.gsub(/Bearer\s+([^\s]*)$/, '\1')
```

Comment:
As is, this returns `""` rather than nil when the `Authorization` header isn't set, which doesn't feel completely right. How about:

```suggestion
    request.headers['Authorization']&.gsub(/Bearer\s+([^\s]*)$/, '\1')
```